README.md

```diff
@@ -2,31 +2,200 @@
+between a pet and their owner, and represents something called a **foreign key**.
+The `owner_id` value should contain an integer that corresponds to an `id` value
+from one of the rows in the `owner` table.  For example, the dog named Wags
+is owned by the owner with id 3, i.e. Dana Abcer.  **Foreign key** references
+are what makes relational databases so powerful in terms of modeling
+real-world entities and their complex relationships.
+
+Imagine how we might connect the idea of a table
+in SQL to a class in Java, and a record within a
+table to an instance of a class.
+For the `owner` table, the Java representation would be an `Owner` class such as the following:
+
+```java
+public class Owner {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String phone;
+}
+```
+
+The Java representation of the `pet` table would be the `Pet` class shown below.  Note
+that the owner is represented as a reference to an instance of the `Owner` class,
+rather than an integer identifier as stored in the `pet` database table:
+
+```java
+public class Pet {
+    private int id;
+    private String name;
+    private String species;
+    private String breed;
+    private int age;
+    private Owner owner;
+}
+```
+
+In a subsequent lesson we will see how to automatically map Java classes
+and database tables, which is called **Object-Relational Mapping** or **ORM**.
+
+## Structured Query Language
+
+**SQL** is a popular query language used with most relational database systems.
+SQL is a declarative language consisting of different types of statements
+for managing data.  The statements are organized into several types,
+each defined by a language subset of SQL:
+
+- DDL – Data Definition Language
+- DQL – Data Query Language
+- DML – Data Manipulation Language
+- DCL – Data Control Language
+- TCL - Transaction Control Language
+
+We will work primarily with DDL, DQL, and DML statements.
+
+### DDL (Data Definition Language)
+
+**Data Definition Language** or **DDL** statements define tables
+and other structural and functional database objects.
+We will primarily work with the following DDL statements:
+
+- **CREATE** - create a database structural or functional object such as a table.
+- **DROP** - delete a database object.
+- **ALTER** - alter a database object.
+- **RENAME** - rename a database object.
+
+For example, we can create the owner table with the following SQL statement:
+
+```sql
+CREATE TABLE owner (
+  id INTEGER PRIMARY KEY,
+  first_name TEXT,
+  last_name TEXT,
+  phone TEXT
+);
+```
+
+### DQL (Data Query Language) 
+
+**Data Query Language** or **DQL** statements are used to perform queries on the data.
+A query selects a subset of rows and/or columns in one or more tables, and may
+perform various functions on the selected data. We will work with one DQL statement:
+
+- **SELECT** - retrieve data from the database.
+
+For example, the following SQL statement will retrieve rows
```

Comment:
Could we clarify this with "will retrieve all rows from the pet table" to emphasize the `*` in the SQL query?

Reply:
updated to explain * shows all columns.

Reply:
Yes - I meant columns not rows lol. Thanks for understanding what I meant!